src/spaceObjects/playerSpaceship.cpp

```diff
@@ -1138,6 +1209,11 @@ void PlayerSpaceship::onReceiveClientCommand(int32_t client_id, sf::Packet& pack
     case CMD_SET_ALERT_LEVEL:
         {
             packet >> alert_level;
+            string alert_level_string = alertLevelToString(alert_level);
```

Comment:
First convert the enum to a string and then check the string? Why not check the enum directly?